lambda/publish-todays-curation/src/curation.ts

```diff
@@ -0,0 +1,132 @@
+import {CopyObjectCommand, HeadObjectCommand, NotFound, S3Client} from "@aws-sdk/client-s3";
+import {format, formatISO} from "date-fns";
+import { sendFastlyPurgeRequestWithRetries } from '@recipes-api/lib/recipes-data';
+import {Bucket, FastlyApiKey} from "./config";
+
+const s3Client = new S3Client({region: process.env["AWS_REGION"]});
+
+export interface CurationPath {
+  region: string;
+  variant: string;
+  year: number;
+  month: number;
+  day: number;
+}
+
+const KnownRegions = [
```

Comment:
As w/ Fronts, this could be 'edition' rather than region, again there may be reasons and it does seem like these map straightforwardly to regions – for now 🌏